pkg/rulesConfig/defaultRules/github/ensure-node-version.json

```diff
@@ -0,0 +1,67 @@
+{
+    "description": "Ensure node-version is set",
+    "failureMessage": "node-version property should be set to a specific version when using action actions/setup-node",
+    "schema": {
+        "type": "object",
+        "additionalProperties": {
+            "type": "object",
+            "properties": {
+                "repositories": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "object",
+                        "properties": {
+                            "github-actions-workflows": {
+                                "type": "object",
+                                "additionalProperties": {
+                                    "properties": {
+                                        "content": {
+                                            "properties": {
+                                                "jobs": {
+                                                    "type": "object",
+                                                    "additionalProperties": {
+                                                        "properties": {
+                                                            "steps": {
+                                                                "type": "array",
+                                                                "items": {
+                                                                    "type": "object",
+                                                                    "if": {
+                                                                        "properties": {
+                                                                            "uses": {
+                                                                                "type": "string",
+                                                                                "pattern": "actions/setup-node*"
```

Comment:
either use `"actions/setup-node"`, and then it should accept whatever contains that (meaning `"actions/setup-node@v2"` should in theory also match), or use `"actions/setup-node.*"`, because current behaviour expects `1 or more e`